tests/unit/src/test/scala/tests/DebugProtocolSuite.scala

```diff
@@ -302,11 +308,11 @@ class DebugProtocolSuite extends BaseLspSuite("debug-protocol") {
         )
         .recover {
           case WorkspaceErrorsException =>
-            server.statusBarHistory
+            WorkspaceErrorsException
```

Comment:
Status bar is actually updated concurrently, which is causing spurious errors in CI